xlators/storage/posix/src/posix-inode-fd-ops.c

```diff
@@ -2529,6 +2570,9 @@ posix_release(xlator_t *this, fd_t *fd)
     VALIDATE_OR_GOTO(this, out);
     VALIDATE_OR_GOTO(fd, out);
 
+    if (fd->inode && fd->inode->fd_count == 0)
```

Comment:
Can we have an 'fd' without an 'inode' ?

Reply:
My bad. Just added that not having the thought behind it